### ISSUE TITLE
Problem with  updating total in claim form

### ISSUE THREAD

**Issue by BusinessTec** [aliomattux/crm_claim_sale_services#1]
Hi,

I am evaluating your module and really like the functionality it offers. I am just fighting some small problems:

The first is an error message while trying to update a total for return: 
AttributeError: 'crm.claim' object has no attribute 'button_dummy'

I am able to save claim form and total is calculated correctly.

**Comment by aliomattux** [comment]
Hi,
I see that the button does not exist as a function. I don't think clients using the module update total so it went unnoticed. If you save a record, doesnt it automatically update the total anyway?

**Comment by BusinessTec** [comment]
Correct, using 'save' option get calculation done  but my instict is always check calculation before saving record and moving on.
